controllers/fenceagentsremediation_controller.go

```diff
@@ -163,18 +170,23 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		if err := utils.RemoveTaint(r.Client, far.Name); err != nil && !apiErrors.IsNotFound(err) {
 			return emptyResult, err
 		}
+		commonEvents.NormalEvent(r.Recorder, node, v1alpha1.EventReasonRemoveNoExecuteTaint, v1alpha1.EventMessageRemoveNoExecuteTaint)
 		// remove finalizer
 		controllerutil.RemoveFinalizer(far, v1alpha1.FARFinalizer)
 		if err := r.Client.Update(context.Background(), far); err != nil {
 			return emptyResult, fmt.Errorf("failed to remove finalizer from CR - %w", err)
 		}
 		r.Log.Info("Finalizer was removed", "CR Name", req.Name)
+		commonEvents.NormalEvent(r.Recorder, far, v1alpha1.EventReasonRemoveFinalizer, v1alpha1.EventMessageRemoveFinalizer)
 		return emptyResult, nil
 	}
 	// Add FAR (medik8s) remediation taint
 	r.Log.Info("Try adding FAR (Medik8s) remediation taint", "Fence Agent", far.Spec.Agent, "Node Name", req.Name)
```

Comment:
At this point, would you consider moving the also the log in the else if branch at line 188?
 Changing the message from "Try to", to "Added taing", of course.

Reply:
Yes. That would be nicer